data/themes/darktable.css

```diff
@@ -143,6 +143,10 @@
 @define-color thumbnail_hover_bg_color @grey_85;
 @define-color thumbnail_hover_fg_color @grey_90;
 @define-color filmstrip_bg_color @darkroom_bg_color;
+@define-color timeline_bg_color @grey50;
+@define-color timeline_fg_color @grey85;
```

Comment:
Same here and I would suggest that line:

`@define-color timeline_fg_color @text_color;`

Reply:
that's even worse as @text_color is @gery_05 but we want @grey_85

Or you want to change completely the timeline color scheme... why not, but I'm not sure that's your goal ;)

Reply:
Changing color scheme is my goal here, yes.